packages/fx-core/src/component/driver/deploy/azure/baseDeployDriver.ts

```diff
@@ -25,7 +25,10 @@ export abstract class BaseDeployDriver extends BaseDeployStepDriver {
     await this.context.logProvider.debug("start deploy process");
 
     const deployArgs = BaseDeployDriver.asDeployArgs(this.args);
-    this.workingDirectory = deployArgs.workingDirectory;
+    // if working dir is not absolute path, then join the path with project path
+    this.workingDirectory = path.isAbsolute(deployArgs.workingDirectory)
+      ? deployArgs.workingDirectory
+      : path.join(this.workingDirectory, deployArgs.workingDirectory);
```

Comment:
Maybe just `path.resolve(this.workingDirectory, deployArgs.workingDirectory)`?

Reply:
I want user can use absolute path in working directory like Zhaofeng did yesterday.